Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ubuntu:22.10
+FROM ubuntu:22.04
 
 RUN apt update && apt install -y build-essential
```

Comment:
You probably want to use the more friendly "apt-get" instead of "apt".

`RUN apt-get -y update && apt-get -y install build-essential`